Build action failing for windows

16s
Run npm run dist

> bottleneck@0.1.14 dist
> npm run build && electron-builder


> bottleneck@0.1.14 build
> npm run build:main && npm run build:preload && npm run build:renderer


> bottleneck@0.1.14 build:main
> tsc -p tsconfig.main.json


> bottleneck@0.1.14 build:preload
> tsc -p tsconfig.preload.json


> bottleneck@0.1.14 build:renderer
> vite build

The CJS build of Vite's Node API is deprecated. See https://vite.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated for more details.
vite v5.4.20 building for production...
transforming...
✓ 2223 modules transformed.
rendering chunks...
[plugin:vite:reporter] [plugin vite:reporter] 
(!) D:/a/bottleneck/bottleneck/src/renderer/services/github.ts is dynamically imported by D:/a/bottleneck/bottleneck/src/renderer/components/conversation/CommentForm.tsx, D:/a/bottleneck/bottleneck/src/renderer/stores/syncStore.ts but also statically imported by D:/a/bottleneck/bottleneck/src/renderer/components/diff/useCommentManager.ts, D:/a/bottleneck/bottleneck/src/renderer/stores/authStore.ts, D:/a/bottleneck/bottleneck/src/renderer/stores/branchStore.ts, D:/a/bottleneck/bottleneck/src/renderer/stores/issueStore.ts, D:/a/bottleneck/bottleneck/src/renderer/stores/prStore.ts, D:/a/bottleneck/bottleneck/src/renderer/views/BranchesView.tsx, D:/a/bottleneck/bottleneck/src/renderer/views/IssueDetailView/useIssueDetailData.ts, D:/a/bottleneck/bottleneck/src/renderer/views/PRDetailView.tsx, D:/a/bottleneck/bottleneck/src/renderer/views/PRListView.tsx, dynamic import will not move module into another chunk.

[plugin:vite:reporter] [plugin vite:reporter] 
(!) D:/a/bottleneck/bottleneck/src/renderer/stores/authStore.ts is dynamically imported by D:/a/bottleneck/bottleneck/src/renderer/stores/syncStore.ts but also statically imported by D:/a/bottleneck/bottleneck/src/renderer/App.tsx, D:/a/bottleneck/bottleneck/src/renderer/components/ConversationTab.tsx, D:/a/bottleneck/bottleneck/src/renderer/components/TopBar.tsx, D:/a/bottleneck/bottleneck/src/renderer/stores/issueStore.ts, D:/a/bottleneck/bottleneck/src/renderer/views/AuthView.tsx, D:/a/bottleneck/bottleneck/src/renderer/views/BranchesView.tsx, D:/a/bottleneck/bottleneck/src/renderer/views/IssueDetailView/useIssueDetailData.ts, D:/a/bottleneck/bottleneck/src/renderer/views/PRDetailView.tsx, D:/a/bottleneck/bottleneck/src/renderer/views/PRListView.tsx, D:/a/bottleneck/bottleneck/src/renderer/views/SettingsView.tsx, dynamic import will not move module into another chunk.

[plugin:vite:reporter] [plugin vite:reporter] 
(!) D:/a/bottleneck/bottleneck/src/renderer/mockData.ts is dynamically imported by D:/a/bottleneck/bottleneck/src/renderer/stores/syncStore.ts but also statically imported by D:/a/bottleneck/bottleneck/src/renderer/stores/issueStore.ts, D:/a/bottleneck/bottleneck/src/renderer/stores/prStore.ts, D:/a/bottleneck/bottleneck/src/renderer/views/IssueDetailView/useIssueDetailData.ts, D:/a/bottleneck/bottleneck/src/renderer/views/PRDetailView.tsx, dynamic import will not move module into another chunk.

computing gzip size...
x Build failed in 8.23s
error during build:
[vite-plugin-moncao-editor] ENOENT: no such file or directory, mkdir 'D:\a\bottleneck\bottleneck\src\renderer\D:\a\bottleneck\bottleneck\dist\renderer'
    at Object.mkdirSync (node:fs:1372:26)
    at Object.writeBundle (D:\a\bottleneck\bottleneck\node_modules\vite-plugin-monaco-editor\dist\index.js:96:20)
    at file:///D:/a/bottleneck/bottleneck/node_modules/rollup/dist/es/shared/node-entry.js:22400:40
    at async Promise.all (index 0)
    at async PluginDriver.hookParallel (file:///D:/a/bottleneck/bottleneck/node_modules/rollup/dist/es/shared/node-entry.js:22310:9)
    at async file:///D:/a/bottleneck/bottleneck/node_modules/rollup/dist/es/shared/node-entry.js:23391:13
    at async catchUnfinishedHookActions (file:///D:/a/bottleneck/bottleneck/node_modules/rollup/dist/es/shared/node-entry.js:22754:16)
    at async build (file:///D:/a/bottleneck/bottleneck/node_modules/vite/dist/node/chunks/dep-D_zLpgQd.js:65714:16)
    at async CAC.<anonymous> (file:///D:/a/bottleneck/bottleneck/node_modules/vite/dist/node/cli.js:829:5)
Error: Process completed with exit code 1.

--- a/vite.config.ts
+++ b/vite.config.ts
@@ -9,13 +9,13 @@ export default defineConfig({
     monacoEditorPlugin({
       languageWorkers: ["editorWorkerService", "typescript", "json"],
       customWorkers: [],
-      publicPath: "./",
+      publicPath: "./monacoeditorwork/",
     }),
   ],
   base: "./",
   root: path.resolve(__dirname, "src/renderer"),
   build: {
-    outDir: path.resolve(__dirname, "dist/renderer"),
+    outDir: "../../dist/renderer",
     emptyOutDir: true,
     sourcemap: false,
     minify: "esbuild",

